Services/data/devicemodels/scripts/FirmwareUpdate-method.js

```diff
@@ -58,7 +58,7 @@ function main(context, previousState, previousProperties) {
     // Reboot - devices goes offline and comes online after 20 seconds
     log("Executing 'FirmwareUpdate' JavaScript method; Firmware version passed:" + context.Firmware);
 
-    var DeviceMethodStatusKey = "DeviceMethodStatus";
+    var DeviceMethodStatusKey = "FirmwareUpdateStatus";
```

Comment:
Suggest renaming the var to FirmwareUpdateStatusKey

Reply:
This key shows up in RM, so it would probably make sense to keep the string as is.